aiopyarr/models/base.py

```diff
@@ -95,12 +83,7 @@ def __init__(
                 if key == ATTR_DATA:
                     value = generate_data(value, datatype)
                 elif key in CONVERT_TO_DATETIME:
-                    if key == "airDateUtc":
-                        value = get_datetime(value, utc=True)
-                    else:
-                        value = get_datetime(value)
```

Comment:
What format are the dates returned from the API in this case? and what does `ciso8601.parse_datetime` do with them?

I assume the logic now is something like:
- if its a utc date, you get a utc date (then assuming .date() wil be correct)
- if its a timesamp aware date, you get a timestamp aware date. (then assuming .date() wil be correct)
- if its no timezone datetime, you get a utc datetime (probably reasonable, then .date() will be correct)